wasm/src/language_service.rs

```diff
@@ -587,3 +613,9 @@ extern "C" {
     )]
     pub type DiagnosticsCallback;
 }
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(typescript_type = "(callables: [string, ILocation][]) => void")]
```

Comment:
Use the `ITestCallable` type you already defined in the other wasm file, instead of the `[string, ILocation]` tuple. Tuples are clunky in TS.